Source/FuseJS/Bundle.uno

```diff
@@ -85,7 +85,6 @@ namespace FuseJS
 				.then(function(list) {
 					//list is an array of paths, such as "assets/image.jpg"
 				})
-			}
```

Comment:
The whole indentation of the .then etc seems off to me. Wouldn't it be cleaner to do
```js
Bundle.list().then(function(list) {
	//list is an array of paths, such as "assets/image.jpg"
})
```
?

Reply:
I agree that would look better. Wanted my fix to be as minimal as possible, but I can change it to what you suggest.